.changelog/unreleased/state-breaking/provider/1340-cryptographic-equivocation-feature.md

```diff
@@ -0,0 +1,3 @@
+- Add the cryptographic verification of equivocation feature to the provider
+  (cf. ADR-005 & ADR-013). ([\#1340](https://github.com/cosmos/interchain-
```

Comment:
Link has the same issue as [above](https://github.com/cosmos/interchain-security/pull/1340/files#r1395465459).